param/serializer.py

```diff
@@ -28,8 +28,36 @@ def schema(cls, pobj, subset=None):
 
     @classmethod
     def serialize_parameters(cls, pobj, subset=None):
+        """
+        Should serialize the parameters on a Parameterized object
```

Comment:
"Should"?

Reply:
Oh, you mean that subclasses should implement that?  I'd prefer to make that explicit:

```suggestion
        Serialize the parameters on a Parameterized object. Must be implemented by subclasses.
```

Reply:
Okay sure, will change that consistently.